receiver/azureblobreceiver/README.md

```diff
@@ -0,0 +1,41 @@
+# Azure Blob Receiver
+
+| Status                   |             |
+| ------------------------ | ----------- |
+| Stability                |[development]|
+| Supported pipeline types | logs,traces |
+| Distributions            | [contrib]   |
+
+
+This receiver reads logs and trace data from [Azure Blob Storage](https://azure.microsoft.com/services/storage/blobs/).
+
+## Configuration
+
+The following settings are required:
+
+- `connection_string:` (no default): Azure Blob Storage connection key, which can be found in the Azure Blob Storage resource on the Azure Portal.
+- `event_hub:`
+  `  end_point:` (no default): Azure Event Hub endpoint triggering on the `Blob Create` event 
```

Comment:
For consistency with existing components this should be `endpoint`, not `end_point`